lib-js/compiler.ts

```diff
@@ -92,7 +92,7 @@ export class DartCompiler implements Compiler {
 
   async compile(path: string, opts: string[]): Promise<Stdio> {
     this.stdin.write(`!cd ${path}\n`)
-    this.stdin.write([...this.initArgs, ...opts].join(" ") + "\n")
+    this.stdin.write(["--verbose", ...this.initArgs, ...opts].join(" ") + "\n")
```

Comment:
shouldn't it also be added in the dart-sass `implArgs` in cli-args.ts, to ensure that dart-sass also runs in verbose mode when running specs against the dart-sass compiled executable rather than the source code ?